docusaurus/docs/reactnative/basics/troubleshooting.mdx

```diff
@@ -246,6 +246,19 @@ dependencies {
 }
 ```
 
+## Blank screen on channel delete
+
+When a channel is deleted, an event is shared from the backend which could be listened to get the status of the deleted channel.
```

Comment:
We should explain in a bit more details about what happens on ChannelList and Channel component, when a channel gets deleted. I mean whats the default behaviour.